### PR TITLE
Update to Golang 1.10.7

### PR DESCRIPTION
**Pull request by ph**
```
go1.10.7 (released 2018/12/14) includes a fix to a bug introduced in Go 1.10.6 that broke go get for import path patterns containing "...". See the Go 1.10.7 milestone on our issue tracker for details.
```

release notes: https://golang.org/doc/devel/release.html#go1.10.minor

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -50,6 +50,7 @@ https://github.com/elastic/beats/compare/v6.5.0...6.x[Check the HEAD diff]
 - Refresh host metadata in add_host_metadata. {pull}9359[9359]
 - When collecting swap metrics for beats telemetry or system metricbeat module handle cases of free swap being bigger than total swap by assuming no swap is being used. {issue}6271[6271] {pull}9383[9383]
 - Update Golang to 1.10.6. {pull}9563[9563]
+- Update Golang to 1.10.7. {pull}9640[9640]
```
Comment (andrewvc) [inline CHANGELOG.asciidoc]: Nit: we can probably delete the 1.10.6 line now.

Also, why are thes elines indented one level deeper than 49? None of this is a blocker for merge however.

Comment (ruflin) [reply CHANGELOG.asciidoc]: Agree

Comment (ph) [reply CHANGELOG.asciidoc]: I've removed the 1.10.6 note, the align is correct, I've witnessed a similar issue with @urso on another PR :(

Comment (ph) [reply CHANGELOG.asciidoc]: This look like a github issue with their way to display diff, if you look at the raw file it is correct. https://raw.githubusercontent.com/elastic/beats/5c208b74f5209c942f8f80186c49a4af01492b2f/CHANGELOG.asciidoc

**Comment by ph** [comment]
Merging, I just fixed the changelog conflict.

**Pull request merged**